modules/cram/templates/samtools_addreplacerg.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -euo pipefail
+
+create_bed () {
+  echo -e "!{bedContent}" > "!{bed}"
+}
+
+create_cram_slice () {
```

Comment:
see https://github.com/molgenis/vip/pull/339: depending on PR merge order a `create_cram_slice _cleanup` should be added